gateway/requests/requests.go

```diff
@@ -62,6 +62,7 @@ type Function struct {
 	Image           string  `json:"image"`
 	InvocationCount float64 `json:"invocationCount"` // TODO: shouldn't this be int64?
 	Replicas        uint64  `json:"replicas"`
+	AvailableCount  int     `json:"availableCount"`
```

Comment:
I might have suggested `AvailableCount` - I wonder if we need to change the name to be cohesive with the `Replicas` value such as ReplicasAvailable / ReplicasReady.